coreprofile/10/jakarta-coreprofile-spec-10.0.html

```diff
@@ -0,0 +1,793 @@
+#footer{padding:0 .9375em}
+.hide-on-print{display:none!important}
+.print-only{display:block!important}
+.hide-for-print{display:none!important}
+.show-for-print{display:inherit!important}}
+@media print,amzn-kf8{#header>h1:first-child{margin-top:1.25rem}
+.sect1{padding:0!important}
+.sect1+.sect1{border:0}
+#footer{background:none}
+#footer-text{color:rgba(0,0,0,.6);font-size:.9em}}
+@media amzn-kf8{#header,#content,#footnotes,#footer{padding:0}}
+</style>
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body class="book toc2 toc-left">
+<div id="header">
+<h1>Jakarta EE CoreProfile</h1>
+<div class="details">
+<span id="author" class="author">Jakarta EE Platform Team, https://projects.eclipse.org/projects/ee4j.jakartaee-platform</span><br>
+<span id="email" class="email"><a href="https://dev.eclipse.org/mailman/listinfo/jakartaee-platform-dev" class="bare">https://dev.eclipse.org/mailman/listinfo/jakartaee-platform-dev</a></span><br>
+<span id="revnumber"> 10.0,</span>
+<span id="revdate">May 16, 2022</span>
+<br><span id="revremark">Final</span>
+</div>
+<div id="toc" class="toc2">
+<div id="toctitle">Table of Contents</div>
+<ul class="sectlevel1">
+<li><a href="#copyright">Copyright</a></li>
+<li><a href="#eclipse-foundation-specification-license">Eclipse Foundation Specification License</a>
+<ul class="sectlevel2">
+<li><a href="#disclaimers">Disclaimers</a></li>
+</ul>
+</li>
+<li><a href="#introduction">1. Introduction</a>
+<ul class="sectlevel2">
+<li><a href="#target-and-rationale-for-the-core-profile">1.1. Target and Rationale for the Core Profile</a></li>
+<li><a href="#determining-applicable-requirements">1.2. Determining Applicable Requirements</a></li>
+<li><a href="#acknowledgements-for-jakarta-ee-10-0">1.3. Acknowledgements for Jakarta EE 10.0</a></li>
+</ul>
+</li>
+<li><a href="#core-profile-definition">2. Core Profile Definition</a>
+<ul class="sectlevel2">
+<li><a href="#required_components">2.1. Required Components</a></li>
+<li><a href="#optional-components">2.2. Optional Components</a></li>
+<li><a href="#additional_requirements">2.3. Additional Requirements</a></li>
+</ul>
+</li>
+<li><a href="#revision-history">Appendix A: Revision History</a>
+<ul class="sectlevel2">
+<li><a href="#changes-in-final-release-ee-10">A.1. Changes in Final Release EE 10</a></li>
+</ul>
+</li>
+<li><a href="#relateddocs">Appendix B: Related Documents</a></li>
+</ul>
+</div>
+</div>
+<div id="content">
+<div id="preamble">
+<div class="sectionbody">
+<div class="literalblock">
+<div class="content">
+<pre>Specification: Jakarta EE CoreProfile
+
+Version: 10.0
+
+Status: Final
+
+Release: May 16, 2022</pre>
+</div>
+</div>
+</div>
+</div>
+<div class="sect1">
+<h2 id="copyright"><a class="anchor" href="#copyright"></a>Copyright</h2>
+<div class="sectionbody">
+<div class="paragraph">
+<p>Copyright (c) 2018, 2022 Eclipse Foundation.</p>
```

Comment:
given the initial draft of this spec document was created in 2021, shouldn't copyright years be set to `2021, 2022` instead? Same for javadoc..

Reply:
Why wouldn't it be when the specification is ratified?

Reply:
Ok, I see for the spec, but the javadoc is based on contents that have exists since Java EE moved to Jakarta, so that would set the original copyright date on the javadoc. We can discuss in the platform call tomorrow.

Reply:
The other strange thing about the current structure of the specs is that the copyright date is actually in the license-efsl.adoc file.